### PR TITLE
Profiling tool: Print UCX and GDS parameters

### PR DESCRIPTION
**Pull request by nartal1**
This is a small PR where we print ucx and gds related paremeters in output file of profiling tool.
Added eventlog which has both ucx and gds parameters for unit testing.
This fixes https://github.com/NVIDIA/spark-rapids/issues/2693

```
+---------------------------------------------------------------+-----------------------------------------------------+
|spark.executorEnv.UCX_ERROR_SIGNALS                            |                                                     |
|spark.executorEnv.UCX_MAX_RNDV_RAILS                           |1                                                    |
|spark.executorEnv.UCX_MEMTYPE_CACHE                            |n                                                    |
|spark.executorEnv.UCX_RC_RX_QUEUE_LEN                          |1024                                                 |
|spark.executorEnv.UCX_RNDV_SCHEME                              |put_zcopy                                            |
|spark.executorEnv.UCX_TLS                                      |cuda_copy,cuda_ipc,rc,tcp                            |
|spark.executorEnv.UCX_UD_RX_QUEUE_LEN                          |1024                                                 |
|spark.rapids.cudfVersionOverride                               |true                                                 |
|spark.rapids.memory.gpu.direct.storage.spill.alignedIO         |false                                                |
|spark.rapids.memory.gpu.direct.storage.spill.alignmentThreshold|8m                                                   |
|spark.rapids.memory.gpu.direct.storage.spill.enabled           |true                                                 |
|spark.rapids.memory.gpu.direct.storage.spill.useHostMemory     |false                                                |
|spark.rapids.memory.gpu.unspill.enabled                        |false                                                |
|spark.rapids.memory.host.spillStorageSize                      |32G                                                  |
|spark.rapids.memory.pinnedPool.size                            |8G                                                   |
|spark.rapids.shuffle.maxMetadataSize                           |512K                                                 |
|spark.rapids.shuffle.transport.enabled                         |true                                                 |
|spark.rapids.shuffle.ucx.bounceBuffers.size                    |8M                                                   |
|spark.rapids.sql.batchSizeBytes                                |1g                                                   |
|spark.rapids.sql.concurrentGpuTasks                            |1                                                    |
|spark.shuffle.manager                                          |com.nvidia.spark.rapids.spark311.RapidsShuffleManager|
|spark.shuffle.service.enabled                                  |false                                                |
+---------------------------------------------------------------+-----------------------------------------------------+

```

### CODE REVIEW

File: tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala [hunk 1]
```diff
@@ -322,4 +322,28 @@ class ApplicationInfoSuite extends FunSuite with Logging {
...
+        EventLogPathProcessor.getEventLogInfo(path, sparkSession).head._1, index)
+      index += 1
+    }
+    assert(apps.size == 1)
+    for (app <- apps) {
+      val rows = app.runQuery(query = app.generateRapidsUcxGdsProperties + " order by key",
+        fileWriter = None).collect()
+      assert(rows.length == 22) // 22 properties captured.
+      // verify  ucx parameters are captured.
+      assert(rows(1)(0).equals("spark.executorEnv.UCX_MAX_RNDV_RAILS"))
```
Comment (tgravescs) [inline tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala:343]: Is order guaranteed here? Would be better to kit reference specific rows

**Review by tgravescs — COMMENTED** [review]
The test file is still pretty big here at 1mb, can we just add these to existing file or have very basic file that just starts with them set and exits, we don’t have to actually run with them

File: tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CollectInformation.scala [hunk 2]
```diff
@@ -98,7 +98,7 @@ class CollectInformation(apps: Seq[ApplicationInfo],
     val messageHeader = "\nSpark Rapids parameters set explicitly:\n"
     for (app <- apps) {
       if (app.allDataFrames.contains(s"propertiesDF_${app.index}")) {
-        app.runQuery(query = app.generateRapidsProperties + " order by key",
+        app.runQuery(query = app.generateRapidsUcxGdsProperties + " order by key",
```
Comment (andygrove) [inline tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CollectInformation.scala:101]: Could we use a more generic name here, such as `generateNvidiaProperties` ?

File: tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala (hunk 1 above)
Comment (nartal1) [reply tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala]: order is guaranteed here. using order by propertyName in the query.

File: tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CollectInformation.scala (hunk 2 above)
Comment (nartal1) [reply tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CollectInformation.scala]: done.

**Comment by nartal1** [comment]
build

**Comment by nartal1** [comment]
@tgravescs I have updated the event log file. Now it's smaller. PTAL.
@andygrove I think I have addressed the review comments. PTAL.

**Review by tgravescs — APPROVED** [review]

**Review by abellina — APPROVED** [review]

**Pull request merged**